### PR TITLE
Completed 257 and re attempted old problems 

### PR DESCRIPTION
**Pull request by jacgit18**

### CODE REVIEW

File: joshuaCarpentier/assignments/heaps/lc295/lc295.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+   * 
+   * Input: 
+   * Output: 
+   *  
+   * Naive Approach
+   * --------------- 
+   * Variables: 
+   * 
+   * 
+   */
```
Comment (knrodriguez) [inline joshuaCarpentier/assignments/heaps/lc295/lc295.js:22]: Missing approach.

File: joshuaCarpentier/assignments/treesBFS/lc107/lc107.js [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+  /**
+   * 
+   * Input: tree
+   * Output: last to root in array
+   *  
+   * Naive Approach
+   * --------------- 
+   * Variables: root, root. left, root.right, array to store the results 
+   * 
+   *
```
Comment (knrodriguez) [inline joshuaCarpentier/assignments/treesBFS/lc107/lc107.js:17]: Missing approach, and edge cases.

File: joshuaCarpentier/assignments/treesBFS/lc107/lc107.js [hunk 3]
```diff
@@ -11,11 +26,32 @@ class TreeNode {
...
+        }
+        console.log(`Checker : ${currentLevel}`);
+        console.log(`Result Check : ${result.unshift(currentLevel)}`);
+
+        // result.unshift(currentLevel);
+    }
+    return result;
+
+  
+  }
```
Comment (knrodriguez) [inline joshuaCarpentier/assignments/treesBFS/lc107/lc107.js:52]: Don't forget to write it twice to build up muscle memory!

File: joshuaCarpentier/assignments/twopointers/lc1/lc1.js [hunk 4]
```diff
@@ -11,26 +11,86 @@
...
+/**
+ * 
+ * Input: array, targetSUm
+ * Output: index numbers that match 9 when added together
+ *  
+ * Naive Approach
+ * --------------- 
+ * Variables: 
+ * 
+ * loop through the array subtract the current[index] the target sum
```
Comment (knrodriguez) [inline joshuaCarpentier/assignments/twopointers/lc1/lc1.js:25]: Then what?

**Review by knrodriguez — APPROVED** [review]

**Comment by knrodriguez** [comment]
Your approach is very sparse, it does not tell me much of how your code solves the prompt. Try to detail out as much as you can what your code will do and how it will solve the prompt. Also write out what the edge cases are and any test cases to test those edge cases.
